pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -114,9 +114,9 @@ func (rc *reconciler) reconciliationLoopFunc() func() {
 						glog.Infof("Started DetachVolume for volume %q from node %q", attachedVolume.VolumeName, attachedVolume.NodeName)
 					}
 					if err != nil &&
-						!goroutinemap.IsAlreadyExists(err) &&
-						!goroutinemap.IsExponentialBackoff(err) {
-						// Ignore goroutinemap.IsAlreadyExists && goroutinemap.IsExponentialBackoff errors, they are expected.
+						!common.IsAlreadyExists(err) &&
+						!common.IsExponentialBackoff(err) {
```

Comment:
@saad-ali @smarterclayton This should also be in the exponentialbackoff package where I think this code ends up. 
I think the gorutine/common is not needed. We can make it simpler.